Sources_v3/StreamChat/Models/Message.swift

```diff
@@ -82,6 +82,9 @@ public struct _ChatMessage<ExtraData: ExtraDataTypes> {
     
     /// A list of users that are mentioned in this message.
     public let mentionedUsers: Set<_ChatUser<ExtraData.User>>
+
+    /// A list of users that participated in this message thread
+    public let threadParticipants: Set<_ChatUser<ExtraData.User>>
```

Comment:
There's this rule of thumb we use when designing model APIs. If the field can have a non-specific number of elements inside, we don't expose it as model objects but only as ids. This is sill not done correctly for some cases (see https://stream-io.atlassian.net/browse/CIS-408?atlOrigin=eyJpIjoiYzg0MzRlMzQyMzNjNDVjOTgxYTNjOTY3MzliMzQ5ZWIiLCJwIjoiaiJ9) but `threadParticipants` is the case where it's better to expose just ids.

Reply:
Will add this to Notion